### ISSUE TITLE
Datatable mobile row expand control isn't read by screenreaders

### ISSUE THREAD

**Issue by jamesjacobs** [jadu/pulsar#185]
Ideally needs swapping to an `a` or `button` with appropriate aria role and label.

Bug confirmed in JAWS and NVDA.

**Comment by Stanton** [comment]
Not strictly a Pulsar issue as mainly down to how the product implements datatables, but Pulsar example(s) should be updated to avoid this issue being introduced.

The control is simply targeted at the element with the `.table-child-toggle` class

Snippet from Pulsar's datatable config:

```json
responsive: {
    details: {
        type: 'column',
        target: '.table-child-toggle'
    }
},
```

In the markup, this is added to a plain icon, this just needs changing to the desired markup.

`{{ html.icon('plus-sign', { 'class': 'table-child-toggle'}) }}`

**Comment by Stanton** [comment]
[Docs](https://jadu.gitbooks.io/pulsar/content/datatable.html) updated to use a link instead of a plain icon
